### DISCUSSION TITLE
part-ii/2018-97/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#98] [Question comments]
Part II 2018 Number Fields: Paper 4, Section II, G 

https://questions.tripos.org/part-ii/2018-97/

**Comment by artie2000** [comment]
(a) is trivial by Eisenstein, but I guess you're not allowed to use it

**Comment by artie2000** [comment]
<details>
<summary>I think the dual basis is</summary>

$$\frac{\sqrt[n]{m}^{m-j}}{nm}$$

</details>

**Reply by dropdbassett** [reply]
I got this too.
